src/test/create-user.test.ts

```diff
@@ -0,0 +1,130 @@
+import axios from 'axios';
+import { expect } from 'chai';
+import { appDataSource } from '../data-source';
+import { User } from '../entity/User';
+import * as bcrypt from 'bcrypt';
+
+describe('Create User Mutation', () => {
+  after(async () => {
```

Comment:
Idealmente, limpamos o banco entre cada teste (`afterEach`) para garantir que um teste não está influenciando no outro.
Com isso, o código de cada it teria o cenário completo, de modo que para entender não precisaria consultar testes anteriores e o funcionamento também não dependeria dos anteriores: o seu teste de usuário repetido, por exemplo, depende do anterior passar e criar o usuário.